algorithms/kernel/dtrees/forest/regression/df_regression_train_dense_default_impl.i

```diff
@@ -589,7 +614,7 @@ bool OrderedRespHelper<algorithmFPType, cpu>::findBestSplitCategoricalFeature(co
     }
     if (bFound)
     {
-        const algorithmFPType impurityDecrease = curImpurity.var - vBest / (isPositive<algorithmFPType, cpu>(totalWeights) ? totalWeights : 1.);
+        const algorithmFPType impurityDecrease = curImpurity.var - vBest / isPositive<algorithmFPType, cpu>(totalWeights) ? totalWeights : 1.;
```

Comment:
reverse changes, otherwise it'll be error on vs2017

Reply:
fixed